paasta_tools/marathon_serviceinit.py

```diff
@@ -149,15 +151,22 @@ def status_marathon_job(service, instance, app_id, normal_instance_count, client
                                      running_instances, normal_instance_count)
 
 
-def get_verbose_status_of_marathon_app(app):
+def get_verbose_status_of_marathon_app(marathon_client, app, service, instance, cluster, soa_dir):
     """Takes a given marathon app object and returns the verbose details
     about the tasks, times, hosts, etc"""
     output = []
     create_datetime = datetime_from_utc_to_local(isodate.parse_datetime(app.version))
     output.append("  Marathon app ID: %s" % PaastaColors.bold(app.id))
     output.append("    App created: %s (%s)" % (str(create_datetime), humanize.naturaltime(create_datetime)))
-    output.append("    Tasks:")
 
+    autoscaling_info = get_autoscaling_info(marathon_client, service, instance, cluster, soa_dir)
```

Comment:
Interesting question. I think in the case where we have two marathon apps the autoscaler will only act on the one that matches the current config+git sha. So really we should probably only print this info on the "new" app...

I think for now I'll leave it as it is, just so we don't block on shipping this. With the change on L375 it shouldn't be too expensive as it's just checking for the presence of min+max instances in cached yaml files.